nfd-daemonset-combined.yaml.template

```diff
@@ -21,10 +21,23 @@ rules:
   - ""
   resources:
   - nodes
+# when using command line flag --resource-labels to create extended resources
+# you will need to uncomment "- nodes/status"
+# - nodes/status
   verbs:
   - get
   - patch
   - update
+  # List only needed for --prune
```

Comment:
We are not deploying nfd-topology-updater in this manifest, why do we need these RBAC rules for noderesourcetopologies

We are calling this manifest combined but I only see nfd-master and nfd-worker? My understanding was that combined would be manifest that would have full funcitonality of NFD (nfd-master + nfd-worker + nfd-topology-updater)

Reply:
> We are not deploying nfd-topology-updater in this manifest, why do we need these RBAC rules for noderesourcetopologies

the one who updates the CRD is the master, no?

> We are calling this manifest combined but I only see nfd-master and nfd-worker? My understanding was that combined would be manifest that would have full funcitonality of NFD (nfd-master + nfd-worker + nfd-topology-updater)

yep I should add here additional container for the topology updated, missed it

Reply:
Sure, master updates the CRD but since we didn't have nfd-topology-updater container in the manifest I assumed that this was a manifest just for nfd-master and nfd-worker. So either we need to add nfd-topology-updater container here or remove the RBAC, I am happy either way :)

Reply:
Added the nfd-topology-updater container